v2/AdminCore.DTOs/Event/EventDateDto.cs

```diff
@@ -4,7 +4,7 @@ namespace AdminCore.DTOs.Event
 {
   public class EventDateDto
   {
-    public int EventDateId { get; set; }
```

Comment:
How come we're going with EventDatesId? Surely this refers to a single event date range?